TheMET/TheMET/FeaturedArtViewController.swift

```diff
@@ -11,4 +11,13 @@ import UIKit
 
 class FeaturedArtViewController: UIViewController {
     
+    override func viewDidLoad() {
+           super.viewDidLoad()
+           
+        let newView = FeaturedArtWorkLoadingView.constructView()
+        newView.awakeFromNib()
```

Comment:
You don't need to call `awakeFromNib` on the view. It is called automatically when it is loaded from XIB

Reply:
fixed